editor/plugins/org.fusesource.ide.wsdl2rest.ui/.classpath

```diff
@@ -21,6 +20,8 @@
 	<classpathentry exported="true" kind="lib" path="libs/commons-digester.jar"/>
 	<classpathentry exported="true" kind="lib" path="libs/commons-logging.jar"/>
 	<classpathentry exported="true" kind="lib" path="libs/commons-validator.jar"/>
+	<classpathentry exported="true" kind="lib" path="libs/commons-lang.jar"/>
```

Comment:
why did we need to add commons-lang in addition to commons-lang3 ?

Reply:
This .classpath was updated to stay coherent with what is described in build.properties. It was missing from when the dependencies were added previously

Reply:
build.properties:
https://github.com/jbosstools/jbosstools-fuse/blob/6dade9464924f3f427e9af9a4b19f6b7d765086b/editor/plugins/org.fusesource.ide.wsdl2rest.ui/build.properties#L104-L105

previous commit adding commons-lang and commons-lang3 to build.properties but only commons-lang3 to .classpath: https://github.com/jbosstools/jbosstools-fuse/commit/35d9f20123022fbc8ce9a2bf3e94f2853ff4c877